src/components/icons/Icon.jsx

```diff
@@ -36,6 +36,7 @@ export type IconTypeType =
   | 'exclamation_mark'
   | 'facebook'
   | 'filters'
+  | 'filters_2'
```

Comment:
maybe we could consider some different name? ie. `funnel`? wdyt?

Reply:
let's wait a bit, I will consult naming. This icon will show up in design tools as well so let's make sure we use the best name :)

Reply:
Yeah, makes sense to me. I focused on the usage in this case (as probably this will be our second option for filters) but I think `funnel` is way better name 😅  @coderitual  ?

Reply:
Let's go with @matzimowski propose - `funnel`

Reply:
i feel very honoured 🥨

Reply:
done ✅